_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -3863,7 +3863,6 @@
 	desc = "A fierce companion for any person of power, this spider has been carefully trained by Nanotrasen specialists. Its beady, staring eyes send shivers down your spine.";
 	emote_hear = list("chitters");
 	faction = list("spiders");
-	harm_intent_damage = 3;
```

Comment:
Why you changing this?

Reply:
Because i'm removing all non-essential use of this var in favor of scaling with species punchdamage